### PR TITLE
Stops maintenance drones from clicking stuff while ventcrawling

### PR DESCRIPTION
**Pull request by SabreML**
## What Does This PR Do

Stops maintenance drones from using any click-based interactions when ventcrawling, since afaik none of them are intentional.
This includes (but is probably not limited to):

**1: Doors
2: Firelocks
3: Fire alarms
4: Fire extinguisher cabinets
5: Light bulbs/tubes
6: Door buttons *(e.g. Medbay/Security Lockdown)*
6.5: Actually all buttons *(Mass driver, toxins/turbine ignition switch, etc)*
7: Robotics bots**

Also adds `is_ventcrawling()` as a define, as shorthand for `(istype(A.loc, /obj/machinery/atmospherics))`.

Also also removes the ability to ctrl-click fire alarms for silicons, since now you can just click normally to toggle it.

*(and technically reverts #13758)*

## Why It's Good For The Game

Bugfixes!

## Changelog
:cl:
del: Removes ctrl-clicking fire alarms for silicons, since you can now just click them to toggle.
fix: Maintenance drones can no longer interact with objects while ventcrawling (e.g. Firelocks, Airlocks, Light bulbs, etc.)
/:cl:

### CODE REVIEW

**Review by SteelSlayer — APPROVED** [review]

File: code/_onclick/cyborg.dm [hunk 1]
```diff
@@ -15,6 +15,8 @@
 		return
 	changeNext_click(1)
 
+	if(is_ventcrawling(src)) // To stop drones interacting with anything while ventcrawling
```
Comment (farie82) [inline code/_onclick/cyborg.dm:18]: This also stops them from examining stuff and what not. Which is fine but it doesn't sound like a fix but more like a tweak

**Review by farie82 — APPROVED** [review]
Code is fine

**Comment by TDSSS** [comment]
Being unable to examine seems kind of annoying.

File: code/_onclick/cyborg.dm (hunk 1 above)
Comment (SabreML) [reply code/_onclick/cyborg.dm:18]: Just tested it on master, and it looks like they can't examine anyway.

**Review by Kyep — APPROVED** [review]

**Pull request merged**